docs/HowTo/authentication/google-account.md

```diff
@@ -36,7 +36,7 @@ in the Klaw-core module to enable the use of Google account login.
 
 5.  Configure an existing SUPERADMIN user from AD to approve new users in the following property for username:
 
-    <klaw.superadmin.default.username=superadmin@company.com>
+    <Klaw.superadmin.default.username=superadmin@company.com>
```

Comment:
Changing spelling of "Klaw" should not be part of this PR, since it's for the rule "Klaw Core", can you reverse this one?